http/headers/Sec-GPC.json

```diff
@@ -12,18 +12,14 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "95",
+              "version_added": "preview",
               "impl_url": "https://hg.mozilla.org/mozilla-central/rev/09e51e6d4a7e",
```

Comment:
Do I need this URLs? No idea if that's still what the URL should be?

Reply:
You don't need them. Especially not after a feature is implemented already.